src/Skyroom/Adapter/WooCommerceAdapter.php

```diff
@@ -316,55 +315,17 @@ function getPostString($plural = false)
     /**
      * @param int $orderId
      * @param \WC_Order $order
-     * @param UserRepository $userRepository
      * @param EventRepository $eventRepository
      */
-    function processOrder($orderId, $order, UserRepository $userRepository, EventRepository $eventRepository)
+    function processOrder($orderId, $order, EventRepository $eventRepository)
```

Comment:
We don't need this hook anymore.

Reply:
No, it's good for the log

Reply:
Yeah it's good. so you can remove try/catch and use a better message for it.